src/TabNavigator.re

```diff
@@ -56,26 +56,39 @@ module type TabConfig = {
   type tabs;
   type order = list(tabs);
   let order: order;
+  let containerName: string;
   let tabBarOptions: tabBarOptions;
   let getTab:
-    tabs => (Js.Dict.key, navigation => ReasonReact.reactElement, screenOptions);
+    (tabs, navigation) =>
+    (Js.Dict.key, unit => ReasonReact.reactElement, screenOptions);
 };
 
 module Create = (Config: TabConfig) => {
   [@bs.deriving abstract]
   type navigatorConfig = {initialRouteName: string};
-  
 
   [@bs.deriving abstract]
   type routeConfig = {
-    screen: navigation => ReasonReact.reactElement,
+    screen: unit => ReasonReact.reactElement,
     navigationOptions: screenOptions,
   };
 
+  module NavigationProp = {
+    [@bs.send] external navigate: string => unit = "navigate";
```

Comment:
Do we pass the Variant to the navigate function, or not?

Reply:
Yeah in the end we need to pass a variant. We don't want to use string. This is just an experiment. This is not working yet. navigation is WIP.

Reply:
Ok, would be great to add a comment there to know what's going there

Reply:
In this case we don't. "NavigationProp" module is a binding to React Navigation's "navigation" that operates on strings and is not aware of ReasonML variant.

This is going to be super similar to https://github.com/callstackincubator/bs-react-navigation/blob/aaf0f84250e35e49bcb0c0ee0281aa038f271bc7/src/StackNavigator.re#L26-L40 and once we complete both, I think there's an opportunity to extract a common part.

Reply:
We should translate variant to string here: https://github.com/callstackincubator/bs-react-navigation/pull/19/files#diff-f5a576ae9729bbd89c5aa8c4a2bf67ffR83